subjects/java/projects/buy-02/audit/README.md

```diff
@@ -0,0 +1,73 @@
+
+###### Did the students convince you with their additions to the database?
+
+##### Review the project repository to check for PRs and code reviews.
+
+###### Are developers following a collaborative development process with PRs and code reviews?
+
+##### Check the implementation of Orders MicroService, User Profile, Search and Filtering, and Shopping Cart functionalities.
+
+###### Are the implemented functionalities consistent with the project instructions?
+
+###### Are the implemented functionalities clean and do they not pop up any errors or warnings in both back and front end?
+
+##### Add products to the shopping cart and refresh the page.
+
+###### Are the added products still in the shopping cart with the selected quantities?
+
+##### Utilize SonarQube to assess code quality and check for improvements based on SonarQube feedback.
+
+###### Are code quality issues identified by SonarQube being addressed and fixed?
+
+##### Review the user interface to ensure it's user-friendly and responsive.
+
+###### Does the application provide a seamless and responsive user experience?
+
+##### Check if proper error handling and validation mechanisms are in place.
+
+###### Are user interactions handled gracefully with appropriate error messages?
+
+##### Verify the implementation of security measures as specified in the project instructions.
+
+###### Are security measures consistently applied throughout the application?
+
+### Collaboration and Development Process
+
+##### Check the repository's PR history and comments to ensure code reviews are conducted.
+
+###### Are code reviews being performed for each PR?
+
+##### Inspect the CI/CD pipeline configuration with Jenkins to ensure automated builds, tests, and deployments.
+
+###### Is the CI/CD pipeline correctly set up and being utilized for PRs?
+
+##### Examine the repository log and PR merges to ensure that branches are being merged as instructed.
+
+###### Are branches merged correctly, and is the main codebase up-to-date?
+
+##### Run a full test of the application to assess functionality and identify any issues.
+
+###### Does the application pass a comprehensive test to ensure that all new features work as expected?
+
+##### Inspect the codebase for unit tests related to different parts of the application.
+
+###### Are there unit tests in place for critical parts of the application?
+
+#### Bonus
+
+##### Verify if the wishlist feature, if implemented, functions correctly.
+
+###### Is the wishlist feature functioning as expected?
+
+##### Check if different payment methods, if implemented, work as intended.
+
+###### Are the implemented payment methods functioning correctly?
+
+This audit checklist provides a comprehensive assessment of the project, ensuring that best practices, code reviews, and testing are followed. Auditors should thoroughly examine each aspect mentioned in the project instructions to assess compliance and quality.
```

Comment:
Did you forget this paragraph or you intentionnaly put it here ?

Reply:
just forget it